src/errors/test-run/templates.js

```diff
@@ -32,8 +32,8 @@ export default {
     `,
 
     [TEST_RUN_ERRORS.pageLoadError]: err => `
-        A request to ${formatUrl(err.url)} has failed.
-        Use quarantine mode to perform additional attempts to execute this test.
+        Failed to load a page at ${formatUrl(err.url)}.
+        Increase the pageRequestTimeout, enable retryTestPages, or use quarantine mode to perform additional attempts to execute this test.
```

Comment:
Is it possible to highlight (apply **bold** or _italic_) to names in CLI output? Like so:

```
 Increase the **pageRequestTimeout**, enable **retryTestPages**, or use...
```